javascript/ql/src/semmle/javascript/dataflow/internal/FlowSteps.qll

```diff
@@ -119,13 +119,22 @@ private module CachedSteps {
   predicate calls(DataFlow::InvokeNode invk, Function f) {
     f = invk.getACallee(0)
     or
+    not f.getTopLevel().isExterns() and
```

Comment:
Does this restriction change the call graph, or is it compensated for somewhere else?

Reply:
Hm, it seems this is a result of my misunderstanding what `getACallee(0)` does - I somehow got the impression that it excludes calls to externs, which isn't the case.

I think I'll revert this change.